src/reducers/characterReducer.js

```diff
@@ -0,0 +1,189 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const initialState = {
+  character: [
```

Comment:
This should be called `characters`, since it's an array. Pay attention to naming and make the names meaningful so it's easier to tell what data the variable or property holds